Assets/scripts/playerMovement.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
```

Comment:
This is fine but you should use New input system instead. The old system is going to be depricated soon.